pkg/cnb/image_rebaser_test.go

```diff
@@ -1,19 +1,19 @@
-package cnb_test
+package cnb
```

Comment:
Did we move this to `cnb` to make `newRemote` private?

Reply:
yes